Lib/ufo2ft/featureWriters/kernFeatureWriter.py

```diff
@@ -337,7 +337,10 @@ def _groupScriptsByTagAndDirection(feaScripts):
         # direction (DFLT is excluded)
         scriptGroups = {}
         for scriptCode, scriptLangSys in feaScripts.items():
-            direction = unicodedata.script_horizontal_direction(scriptCode)
+            if scriptCode:
+                direction = unicodedata.script_horizontal_direction(scriptCode)
```

Comment:
or you could also do `direction = unicodedata.script_horizontal_direction(scriptCode, default="LTR")`

Reply:
actually that may hide other bugs, it's better like you did (we still get a KeyError if a not-None scriptCode is not found by script_horizontal_direction)